src/tests/ftest/slurm_setup.py

```diff
@@ -264,6 +295,10 @@ def main():
         "-s", "--sudo",
         action="store_true",
         help="Run all commands with privileges")
+    parser.add_argument(
```

Comment:
Shouldn't this new argument be used in CI by default by adding it to https://github.com/daos-stack/daos/blob/master/src/tests/ftest/scripts/main.sh#L216